uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-record-actions.component.js

```diff
@@ -7,7 +7,6 @@ define([
 ], function (components) {
   components.component('leaveRequestRecordActions', {
     bindings: {
-      btnClass: '@',
```

Comment:
why are we removing this? I see that we are still using the bindings in the html

Reply:
There are now two buttons, blue + grey, as per the requirement. But you are right, the bindings still use that, let me remove.

Reply:
Bindings removed. We now have blue + grey everywhere, even in Admin Dashboard.